### ISSUE TITLE
Feedback for me

### ISSUE THREAD

**Issue by djc** [djc/askama#1070]
@Kijewski @GuillaumeGomez I was sorry to see you close your open PRs recently, and wanted to say explicitly that I've very much appreciated all of your contributions to Askama.

I'm assuming the spur of closing PRs is related to my comment in https://github.com/djc/askama/pull/1036#issuecomment-2173018216. I would describe my stance in that PR as prioritizing maintainability over usability of the documentation. Reasonable can disagree about this trade-off -- while I value the improvement of documentation (in terms of maintaining per-version documentation) I worry about my/our ability to maintain Python code in the long term. I don't enjoy maintaining Python code because I feel that it is generally pretty brittle compared to Rust code, and so keeping that code in this repository makes this project harder to maintain.

For each of the PRs that got closed, I could explain why I have not followed up sooner/why I gave the feedback I gave, and generally I stand by the content of the feedback I've given. Obviously I'm not sure why you closed these PRs, although I'm going to assume it is out of some form of dissatisfaction or frustration with my feedback or the lack of (timely) follow-up from my side.

I'm currently very busy with my day job, stuff going on with my children and other open source projects that are more important to me right now, so for some of these difficult trade-offs I haven't had the energy to address them. In some cases I choose not to give feedback at all because I couldn't find the time/energy to give feedback of the kind I strive to give, and even then I realize my feedback can be direct, be extremely detail-oriented and focus on potential for improvements.

If you're willing, I'd be happy to hear more about why you decide to close these PRs and if/how you want to continue to contribute to Askama (we could also discuss how I could do better). If you prefer not to, that's obviously fine too -- I'm very grateful for your constributions so far.

PRs that got closed:

- #1001
- #1023 
- #1036
- #1054 
- #1055 
- #1058

**Comment by GuillaumeGomez** [comment]
Overall, I greatly appreciated your code reviews, they're precise and often very insightful.

In my case, I spent way too much time on #1058, not on the feature itself but on making the style looks like you expected. Like I said, if there is no check I can run to "fix" it, then it's just a huge time loss because I need to wait for you to review and then come back to it.

Another important point is that changes are very slow to be reviewed/merged. It's not in itself a big issue, but added to what I explained above, it's just too much time and energy.

I don't mind reviewing PRs but I don't intend writing new ones.

Also to be transparent with you, I created the `rinja-rs` github organization and the [rinja](https://github.com/rinja-rs/rinja/) repository which is a fork of `askama` because I still strongly believe in this project (and still need it for docs.rs and rustdoc). All the closed PRs were merged there and work is moving quite steadily for now. We'll see where we go from this point.

Again, and I insist: it's mostly because the amount of time and energy it requires from me to contribute on `askama` is too high to be worth it anymore, nothing against. You did an amazing job on `askama` and I hope it'll continue to strive.
